ui/apps/lists/src/extensions/entry-card-save-button.tsx

```diff
@@ -60,13 +63,13 @@ const EntryCardSaveButton = (props: RootExtensionProps<CardSaveButtonExtensionDa
 
   return (
     <Stack align="center">
-      <button onClick={handleEntryBookmark}>
+      <Button onClick={handleEntryBookmark}>
```

Comment:
could you confirm this button works as intended? we have icon button to simplify this block, also if you need to just have the onclick, you may use the plain prop